components/identity-core/org.wso2.carbon.identity.base/src/main/java/org/wso2/carbon/identity/base/AuthenticatorPropertyConstants.java

```diff
@@ -28,4 +28,15 @@ public static enum DefinedByType {
         SYSTEM,
         USER
     }
+
+    /**
+     * The Authentication Type -
+     * Identification: Can collect the user identifier and authenticator. Can engage in any step of the login flow.
```

Comment:
Can collect the user identifier, credential and authenticate.

Reply:
Addressed with commit c4009b8301f34d84dcfd97dfe775da67c9fd14d8